### PR TITLE
Added Sindhi support

### PR DESCRIPTION
**Pull request by fadkeabhi**
Added translation for Sindhi language

### CODE REVIEW

**Review by p-fernandez — CHANGES_REQUESTED** [review]
Thanks for your PR! There is one change missing to be able to approve this. An entry for the new language is missing in this file:
[docs/docs/notification-center/react-components.md
](https://github.com/novuhq/novu/blob/bd7c2490725f031cb832ab8d801ad61ae9e293fc/docs/docs/notification-center/react-components.md?plain=1#L196)
Could you please add it? Thank you.

**Comment by fadkeabhi** [comment]
added entry for language in [docs/docs/notification-center/react-components.md](https://github.com/novuhq/novu/blob/bd7c2490725f031cb832ab8d801ad61ae9e293fc/docs/docs/notification-center/react-components.md?plain=1#L196)

**Review by scopsy — APPROVED** [review]
Thank you so much for your contribution! Really appreciate it 🙏

**Comment by fadkeabhi** [comment]
@p-fernandez I have made changes addressed by you. Please review them.

**Review by p-fernandez — APPROVED** [review]

File: packages/notification-center/src/i18n/lang.ts [hunk 1]
```diff
@@ -104,6 +106,9 @@ export type I18NLanguage =
   | 'ne'
   | 'pl'
   | 'cs'
+  | 'pu'
+  | 'ta'
```
[the remark this answers is not in the document]
Comment (fadkeabhi) [reply packages/notification-center/src/i18n/lang.ts]: done as you said

**Review by scopsy — APPROVED** [review]

**Pull request merged**